packages/app/src/runner/aut-iframe.ts

```diff
@@ -426,40 +477,6 @@ export class AutIframe {
     })
   }
 
-  beforeScreenshot = (config) => {
```

Comment:
original in [runner-shared](https://github.com/cypress-io/cypress/pull/18075/files#diff-e79a069c075dafc43ba805a69ef368962f7fc50f54a07476b039dd0c470f04c0L397) in `9.x`